app/backend/forum/models.py

```diff
@@ -0,0 +1,28 @@
+from backend.models import CustomUser
+from django.db import models
+
+# Create your models here.
+
+class Post(models.Model):
+    title = models.CharField(max_length=100, null=False)
+    author = models.ForeignKey(CustomUser, null=False, on_delete=models.CASCADE)
+    body = models.TextField(null=False)
+    date = models.DateTimeField()
+    upvote = models.IntegerField(null=False, default=0)
+    downvote = models.IntegerField(null=False, default=0)
+
+class PostImages(models.Model):
+    image_url = models.CharField(max_length=100)
```

Comment:
I think 100 max length is not enough.

Reply:
> I think 100 max length may not be enough.

It is probably enough but to guarantee it, I will increase the maximum length constraint.